### ISSUE TITLE
Chrome 57 update breaks bitmap fill rendering

### ISSUE THREAD

**Issue by macropp** [CreateJS/EaselJS#857]
Hi,
We have several games which use Adobe Animate to export the graphics and then we extend the exported classes and make use of them using JS. We use spritesheets when exporting graphics.

After Chrome 57 update we've observed that some of the sprites are not being rendered at all and stage color is seen instead of them. 

Tracking the problem we've seen that it happens on sprites which use bitmap fills. Do you have any work-arounds for this? How should we proceed?

**Comment by endel** [comment]
One problem I noticed that started to occur in Chrome 57 is that `image.naturalWidth` is not immediately available.

**Example:**   (https://jsfiddle.net/endel/qd06sxw7/)

```javascript
var img = document.createElement("img");
img.src = canvas.toDataURL("image/png");
console.log(img.naturalWidth) // => 0
```

This can be a problem particularly when using [`SpriteSheetUtils.extractFrame`](https://github.com/CreateJS/EaselJS/blob/70f2d23a1b0ca4a270c43a3fe2c662d905eeb82a/src/easeljs/utils/SpriteSheetUtils.js#L123-L137
).

**Comment by endel** [comment]
This means [`displayObject.getBounds()`](https://github.com/CreateJS/EaselJS/blob/70f2d23a1b0ca4a270c43a3fe2c662d905eeb82a/src/easeljs/display/Bitmap.js#L199) may not return the value you're expecting.

**Comment by macropp** [comment]
@endel That may exactly be the problem, but we're using Adobe Animate to export all our symbols, so we can't go and change all the generated code..

**Comment by perdedork** [comment]
I'm not sure this will help, but by searching through the Animate generated .js file for this:
`this.shape.graphics.bf(cjs.SpriteSheetUtils.extractFrame(ss`
I identified all of the shapes that had bitmaps.

I found all of those shapes in Animate, and converted them to bitmaps by using the right-click menu "Convert to Bitmap".
I was able to get all of the stuff that Chrome broke to show back up again.
